content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Change Your Container Runtime from Docker Engine to containerd"
+weight: 8
+content_type: task 
+---
+
+- Drain Node 
+```
+# replace <node-to-drain> with the name of your node you are draining
+kubectl drain <node-to-drain> --ignore-daemonsets
+```
+- Stop the Docker daemon
+```
+systemctl stop kubelet
+systemctl stop docker
+```
+- Install & start containerd 
+
+
+## Migrate a Linux node {#migration-linux}
+Use the following commands to install Containerd on your system:
+
+Install and configure prerequisites:
+
+```shell
+cat <<EOF | sudo tee /etc/modules-load.d/containerd.conf
+overlay
+br_netfilter
+EOF
+
+sudo modprobe overlay
+sudo modprobe br_netfilter
+
+# Setup required sysctl params, these persist across reboots.
+cat <<EOF | sudo tee /etc/sysctl.d/99-kubernetes-cri.conf
+net.bridge.bridge-nf-call-iptables  = 1
+net.ipv4.ip_forward                 = 1
+net.bridge.bridge-nf-call-ip6tables = 1
+EOF
+
+# Apply sysctl params without reboot
+sudo sysctl --system
+```
```

Comment:
Can we break each one of these chunks into their own steps? 

(sorry, I'd do this as a suggestion but it's breaking GitHub's formatting!)

2. SOME DESCRIPTIVE ACTION STATEMENT

```shell
cat <<EOF | sudo tee /etc/modules-load.d/containerd.conf
overlay
br_netfilter
EOF
```

3. SOMETHING ELSE DESCRIPTIVE

```shell
sudo modprobe overlay
sudo modprobe br_netfilter
```

4. Setup required sysctl params which persist across reboots

```shell
cat <<EOF | sudo tee /etc/sysctl.d/99-kubernetes-cri.conf
net.bridge.bridge-nf-call-iptables  = 1
net.ipv4.ip_forward                 = 1
net.bridge.bridge-nf-call-ip6tables = 1
EOF
```

5. Apply sysctl params without reboot

```shell
sudo sysctl --system
```